### PR TITLE
Add additional testing

### PR DESCRIPTION
**Pull request by jasalisbury**
https://github.com/department-of-veterans-affairs/vets-contrib/issues/3133
 
* Fixed some failing tests in `plugins.js`
* Additional fields in a route at the same depth of `name`, `id` and `attributes` are ignored. Adding test to verify

### CODE REVIEW

File: test/plugins.test.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ describe("plugins", () => {
                 'config.foo': "bar"
             }
         }])
-        .map(x => x({hasGlobalPlugin: () => false}));
+        .map(x => x({
+            getVersion: () => '1.0.0',
```
Comment (dvogel) [inline test/plugins.test.js:25]: What is the significance of this version value? Where is it coming from?

Comment (jasalisbury) [reply test/plugins.test.js:25]: Was picked after looking where getVersion is [used](https://github.com/adhocteam/kongfig/blob/master/src/core.js#L697). I can add a comment explaining where I got it and how the value doesn't have any impact on the tests.

Comment (dvogel) [reply test/plugins.test.js:25]: A comment with a reference to that linked code would be great.

**Review by kalilsn — APPROVED** [review]
I'm on board with this, although to me it seems like the most likely reason for this breaking would be kong adding an actual `skip_auth` field, which this kind of test wouldn't catch. Maybe that's just paranoid, but if we're going to keep adding things to the kongfig files that shouldn't make it to kong itself, perhaps we should add a prefix to them?

**Comment by dvogel** [comment]
> I'm on board with this, although to me it seems like the most likely reason for this breaking would be kong adding an actual `skip_auth` field, which this kind of test wouldn't catch. Maybe that's just paranoid, but if we're going to keep adding things to the kongfig files that shouldn't make it to kong itself, perhaps we should add a prefix to them?

I could get behind a prefix.

**Review by kalilsn — APPROVED** [review]

**Comment by jasalisbury** [comment]
This PR is not needed, we are going to have an explicit list of services excluded from the auth check instead of impeding that info inside the config.

**Pull request closed without merging**